src/styles/blocks/catalog.scss

```
@@ -17,6 +17,12 @@
     border: 1px solid #ccc;
     border-radius: 5px;
     margin-bottom: 48px;
+    transition: color 0.3s ease-in-out, transform 0.3s ease-in-out;
```

You can also set it like this if you have many properties with same transition
```suggestion
    transition-property: color, transform;
    transition-duration: 0.3s;
    transition-timing-function: ease-in-out;
```